### ISSUE TITLE
why the input_dim becomes FLAGS_hidden1*2?

### ISSUE THREAD

**Issue by Satonio1** [google-research/google-research#104]
I have the problem with the code below? why the input_dim becomes FLAGS_hidden1*2?the input_dim should equal to the last layer's output_dim??

for _ in range(self.num_layers - 2):
      self.layers.append(
          layers.GraphConvolution(
              input_dim=FLAGS.hidden1 * 2,
              output_dim=FLAGS.hidden1,
              placeholders=self.placeholders,
              act=tf.nn.relu,
              dropout=True,
              sparse_inputs=False,
              logging=self.logging,
              norm=self.norm,
              precalc=False))

**Comment by pyyush** [comment]
train_feats = np.hstack((train_feats, feats))
changes N x 100 to N x 200, so all further dims need to be changed.

**Comment by lygztq** [comment]
They concatenate the input features of the GCN layer and features obtained after aggregation
